chart/values.yaml

```diff
@@ -60,23 +60,24 @@ diagnosticMode:
 ## @section CARTO common parameters
 ##
 
-## @param onPremDomain CARTO on premises domain
+## @param selfHostedDomain CARTO Self Hosted domain
 ##
-onPremDomain: ""
+selfHostedDomain: "<FILL_ME>"
+
 
 ## @param commonConfiguration [object] Configuration script that will be run in all Carto instances
 ##
 commonConfiguration: |
   # CARTO Self hosted configuration
-
+  
   # Version
-  CARTO_ONPREMISE_VERSION="{{ .Chart.AppVersion }}"
+  CARTO_SELFHOSTED_VERSION="2022.02.10"
```

Comment:
No, please, leave the previous value and change the version in `chart/Chart.yaml` . `AppVersion` to remain in the helm standard.